MANUAL.txt

```diff
@@ -4168,6 +4168,8 @@ ordered list marker in place of a numeral:
     #. one
     #. two
 
+Note:  the `fancy_lists` extension doesn't work with `commonmark`.
```

Comment:
I think this is a bit too general, as only the `#.` syntax doesn't work. Other features, like using `(a)`, `(b)` as item markers, *do* work.

Reply:
Thanks. I updated the note.